templates/dashboard/base.html

```diff
@@ -2,13 +2,21 @@
 {% load staticfiles i18n %}
 {% load render_bundle from webpack_loader %}
 
-<html class="no-js">
+<html lang="{{ LANGUAGE_CODE }}" class="no-js">
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
 
-    <title>{% block title %}{% trans "Dashboard - Saleor" %}{% endblock %}</title>
+    <title>{% block title %}{% trans "Dashboard - Saleor" context "Dashboard default page title"  %}{% endblock %}</title>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
+      ga('create', 'UA-10159761-19', 'auto');
+      ga('send', 'pageview');
+    </script>
     {% render_bundle 'vendor' 'css' %}
```

Comment:
I had to double check if theres any other difference than readonly banner. Done.